src/System.Runtime/tests/Performance/Perf.Int32.cs

```diff
@@ -62,16 +63,22 @@ public void TryFormat(int value)
 
         [Benchmark(InnerIterationCount = InnerCount)]
         [MemberData(nameof(Int32Values))]
-        public void Parse(int value)
+        public void ParseInteger(int value) => Parse(value, null, NumberStyles.Integer);
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [MemberData(nameof(Int32Values))]
+        public void ParseHex(int value) => Parse(value, "X", NumberStyles.HexNumber);
```

Comment:
As long as you are doing `X` should you also add for the `x` case? If I recall you were going to do some optimization for checking the casing of the larger hex values.